systemtest/src/test/java/io/strimzi/systemtest/StrimziUpgradeST.java

```diff
@@ -57,27 +57,27 @@ private void deleteInstalledYamls(File root) {
     }
 
     @Test
-    void upgrade_0_8_2_to_HEAD() throws IOException {
+    void upgrade_0_10_0_to_HEAD() throws IOException {
```

Comment:
Do we really want to remove upgrade from older versions and test only new ones? We can easily duplicate test and just download different versions and in some cases do some additional steps. @serrss  WDYT?

Reply:
Don't just copy+paste the test, but factor out a common method which takes the version (and expected zk images name) as parameter.

Reply:
Maybe we can try to create parametrized test as it is for standalone Kafka.

Reply:
@tombentley @Frawless This PR has a fix the failure and I just updated version for more relevant. To have an upgrading for more Strimzi versions I opened Github issue #1441.

Reply:
I we also need to care about Kafka version in #1441

Reply:
finally, we decided to have this test as parametrized and test only major versions